api/src/main/java/com/messagebird/objects/conversations/ConversationWebhookBaseRequest.java

```diff
@@ -0,0 +1,41 @@
+package com.messagebird.objects.conversations;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Contains common fields for webhook requests.
+ */
+public abstract class ConversationWebhookBaseRequest {
```

Comment:
Although the idea is nice and it works well now, I'm not a fan of having a shared base class for the create and update endpoints. They're separate endpoints, so imho they should be completely separate in the SDK as well. If the API changes in the future, this would also force us to publish a new major version of this SDK. What do you think?

Reply:
For me, using a base class for creating and updating webhook endpoints makes sense in here because those endpoints use the same domain object.